repl/src/test/scala/org/apache/spark/repl/IntSumUdf.scala

```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+package org.apache.spark.repl
```

Comment:
So, `IntSumUdf.scala` is moved from `resources` to `test` directory?

Reply:
yes, so that it will get compiled.